### ISSUE TITLE
Cannot access login page

### ISSUE THREAD

**Issue by fwalloe** [HubPress/hubpress.io#347]
I can no longer access the log in-page after making a range of changes today. 

When I try to access the page, the wheel in the middle of the page appears and spins, but the actual login page never finishes loading. 

repository: fwalloe/infosecbriefly
In config.json: "repositoryName":"infosecbriefly" 

Some additional details:
1. I am not using the master branch (already got something else there)
2. I changed the name of the repository earlier today
3. I have deleted the tokens, but that did not solve the problem.

**Comment by anthonny** [comment]
Hi @fwalloe 

It looks good for me:
<img width="1012" alt="capture d ecran 2016-03-22 a 21 45 30" src="https://cloud.githubusercontent.com/assets/2006548/13967457/86b457be-f077-11e5-8b41-bfdef4d35761.png">

Can you tell me more? Maybe screenshot with the console

**Comment by fwalloe** [comment]
Now it works for me too. No idea what fixed it, but thanks for the reply

**Comment by JacksonBates** [comment]
I had a similar problem and this was the first hit on Google, so I'll post my solution in case it helps others.

I cleared my browser cache for the last 24 hours of activity (I installed hp last night, so I cleaned up anything from hp, ymmv), and then everything worked normally.

I _suspect_ that I caused the problem in the first place by attempting to change a site setting from an open browser window that had timed out and I hadn't reauthenticated, but I haven't tested that theory.
